lang/ruby/lib/avro/schema_validator.rb

```diff
@@ -183,6 +187,10 @@ def ruby_to_avro_type(ruby_class)
           Hash => 'record'
```

Comment:
Yes. I sort of made the (hidden) mental leap to `.is_a?` as it would work in all versions.

```diff
-        avro_type = if value.class == Integer
+        avro_type = if value.is_a? Integer
```